apps/src/utils.js

```diff
@@ -128,6 +131,7 @@ Function.prototype.inherits = function (parent) {
  * done so that level builders can specify required blocks with wildcard fields.
  */
 export function wrapNumberValidatorsForLevelBuilder() {
+  declare var Blockly: Object;
```

Comment:
Flow's [documentation seems to suggest](https://flowtype.org/docs/declarations.html#declaring-definitions-that-should-exist-at-run-time) that this will work, but then illustrate that it won't.  Any idea what's going on here?

![screenshot from 2016-08-30 12-37-01](https://cloud.githubusercontent.com/assets/1615761/18104147/933fcfbc-6eae-11e6-8730-ad47b0f0e1f8.png)

I have to admit that I like comment annotations better for stuff like this.  It'd be really easy for me to see this line and assume we were shadowing the global `Blockly`, which would significantly change the meaning of this function.